applications/vanilla/settings/stub.php

```diff
@@ -27,58 +20,81 @@
    <li>Bookmark a discussion (click the star) to get notifications for new comments. You can edit notification settings from your profile.</li>
 </ul>
 Go ahead and edit or delete this discussion, then spread the word to get this place cooking. Cheers!";
-$CommentBody = "This is the first comment on your site and it&rsquo;s an important one.
```

Comment:
This is not proper vanilla html formatting and inserts `<br/>` tags into the content. Really, the stub content should be markdown.